event/extractor_noop.go

```diff
@@ -10,6 +10,6 @@ func NewNoopExtractor() Extractor {
 	return &noopExtractor{}
 }
 
-func (s *noopExtractor) Extract(t model.ProcessedTrace) []*model.APMEvent {
-	return nil
+func (e *noopExtractor) Extract(s *model.WeightedSpan, priority int) (extract bool, rate float64) {
```

Comment:
You can also do:
```suggestion
func (*noopExtractor) Extract(_ *model.WeightedSpan, _ int) (extract bool, rate float64) {
```
Just FYI.